packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -120,46 +206,52 @@ def reduce_logging():
 
 @pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
 def test_vip(dcos_api_session, reduce_logging):
-    """Test every permutation of VIP
-    """
+    '''Test every permutation of VIP
```

Comment:
Would be nice to have a comment here explaining the different options for docker and mesos containers, and the permutations we are aiming to test. For someone not familiar with our networking semantics this would be pretty incomprehensible.